src/tests/gov/nasa/jpf/test/java/util/LoggerTest.java

```diff
@@ -0,0 +1,18 @@
+package gov.nasa.jpf.test.java.util;
+
+import gov.nasa.jpf.util.test.TestJPF;
+import org.junit.Test;
+import java.util.logging.Logger;
+
+public class LoggerTest extends TestJPF {
+
+    @Test
+    public void testGetLogger() {
+        if (verifyNoPropertyViolation()) {
+            System.out.println("--- calling getLogger");
+            Logger log = Logger.getLogger("testLogger");
+            assertNotNull(log);
+            System.out.println("--- success: " + log.getName());
```

Comment:
Change this statement to an assertion (e.g., `assertEquals(log.getName(), "testLogger"`) and add a few assertions to check other log settings.